js-objects-and-arrays/js-objects-and-arrays_arrays/index.js

```diff
@@ -3,51 +3,56 @@ console.clear();
 // EXERCISE 1
 // Modify the array exampleArray so that it contains a number and a string.
 
-const exampleArray = [true];
+const exampleArray = [25, "hello"];
```

Comment:
I think the approach would be to use some in-built array methods in order to replace the value it currently holds? For example: push/splice combination?

Reply:
This is the very first assignment, I think it's just on understanding what kinds of data can be added to an array. I agree that it is better to use something like : const exampleArray = [true];
exampleArray[0] = "Some string";
exampleArray.push(100); But it's fine for the first challenge.